src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -56,7 +52,7 @@
                     :emoji                  emoji
                     :emoji-background-color channel-color
                     :on-enter-channel       (fn []
-                                              (utils/show-popup
+                                              (js/alert
```

Comment:
why ?

Reply:
that dependency for utils came from status_im folder and this is just mock data showing a basic case of this functionality.